README.md

```diff
@@ -51,18 +53,26 @@ conda install -c conda-forge jupyterlab
 
 #### pip
 
+pip is a package management system for installing and updating Python packages. pip comes with Python, so you get pip simply by installing Python.  On Ubuntu and Fedora Linux, you can simply use your package manager to install the `python3-pip` package.  [*The Hitchhiker's Guide to Python* provides some guidance on how to install Python on your system if it isn't already](https://docs.python-guide.org/starting/installation/); you can also [install Python directly from python.org](https://www.python.org/getit/).  You might want to [upgrade pip](https://pip.pypa.io/en/stable/installing/) before using it to install other programs.
+
+JupyterLab is implemented in Python, and as of 2018-07 JupyterLab can run on either Python2 or Python3.  However, it's recommended that you install JupyterLab on Python3, even when running Python2 code.  Thus, it's recommended that you install Python3, and then use `pip3` to install JupyterLab (`pip3` is the version of `pip` that always uses Python3; in many cases the command `pip` will use Python2, and that is not recommended).
```

Comment:
Thanks for this PR, the overall language and links are very helpful.  I use Anaconda on both MacOSX and Windows, and neither provides the `pip3` command by default.  Perhaps we should say that if your installation has the `pip3` command, substitute it for the use of `pip` below.  On the other hand, I do have a `python3` executable, so perhaps `python3 -m pip` is the most universal way.